charts/argo-workflows/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: argo-workflows
 description: A Helm chart for Argo Workflows
 type: application
-version: 0.13.1
+version: 0.13.2
```

Let's bump minor version as this PR adds a feature.
```suggestion
version: 0.14.0
```